apps/api/src/app/shared/services/distributed-lock/distributed-lock.service.spec.ts

```diff
@@ -10,6 +10,8 @@ import { DistributedLockService } from './distributed-lock.service';
 
 const originalRedisCacheServiceHost = process.env.REDIS_CACHE_SERVICE_HOST;
 const originalRedisCacheServicePort = process.env.REDIS_CACHE_SERVICE_PORT;
+const originalRedisClusterServiceHost = process.env.REDIS_CLUSTER_SERVICE_HOST;
+const originalRedisClusterServicePorts = process.env.REDIS_CLUSTER_SERVICE_PORTS;
```

Comment:
Should this be?
```suggestion
const originalRedisClusterServicePorts = process.env.REDIS_CLUSTER_SERVICE_PORT;
```

Reply:
No, the name is right as this is for the case where we provide the Redis Cluster with the ports nodes into an array. Nothing to do with the Elasticache cluster set up. This is the fallback.